### ISSUE TITLE
ReplaceDNSRecordsWithTypeAndName

### ISSUE THREAD

**Issue by taiyoungjang** [ahwm/GodaddyWrapper.Net#19]
ReplaceDNSRecordsWithTypeAndName
error is not a array
twice called JsonConvert.SerializeObject() 
maybe change to next code.
```c#
namespace GodaddyWrapper.Helper
{
    internal static class HttpClientExtensions
    {
        public static Task<HttpResponseMessage> PostAsync(this HttpClient client, string requestUri, string jsonString)
        {
            return client.PostAsync(requestUri, new StringContent(jsonString, Encoding.UTF8, "application/json"));
        }

        public static Task<HttpResponseMessage> PutAsync(this HttpClient client, string requestUri, string jsonString)
        {
	  return client.PutAsync(requestUri, new StringContent(jsonString, Encoding.UTF8, "application/json"));
        }

        public static Task<HttpResponseMessage> PatchAsync(this HttpClient client, string requestUri, string jsonString)
        {
            var method = new HttpMethod("PATCH");
            var request = new HttpRequestMessage(method, requestUri){
                Content = new StringContent(jsonString, Encoding.UTF8, "application/json")
            };
            HttpResponseMessage response = new HttpResponseMessage();
            return client.SendAsync(request);
        }
    }
}
```

**Comment by ahwm** [comment]
Nice catch. Thanks.

**Comment by ahwm** [comment]
Fixed in 1.2.4
